src/XRP/issued-currency-client.ts

```diff
@@ -704,10 +760,15 @@ export default class IssuedCurrencyClient {
    * @param sender The Wallet from which issued currency will be sent, and that will sign the transaction.
    * @param destination The destination address (recipient) for the payment, encoded as an X-address (see https://xrpaddress.info/).
    * @param currency The three-letter currency code of the issued currency being sent.
-   * @param issuer The issuing address of the issued currency being sent.
-   * @param amount The amount of issued currency to send.
-   * @param transferFee Optional - can be omitted if sender address or destinationAddress are the same address as the issuer.
-   *                    The transfer fee associated with the issuing account, expressed as a percentage. (i.e. a value of .5 indicates a 0.5% transfer fee).
+   * @param issuer The issuer to specify in the destination amount field of the payment, encoded as an X-address.
+   *               Typically (but not always) the original issuer of the currency.  See https://xrpl.org/payment.html#special-issuer-values-for-sendmax-and-amount.
+   * @param amount The amount of issued currency to pay to the destination.
+   * @param transferFee (Optional) The transfer fee associated with the issuing account, expressed as a percentage. (i.e. a value of .5 indicates
+   *                    a 0.5% transfer fee).  Supply this field for automatic calculation of the sendMax value for this payment.
+   *                    Either this or sendMaxvalue may be specified, but not both.
+   * @param sendMaxValue (Optional) A manual specification of the maximum amount of source currency this payment is allowed to cost,
+   *                      including transfer fees, exchange rates, and slippage. Does not include the XRP destroyed as a cost for submitting \
+   *                      the transaction. Either this or transferFee may be specified, but not both.
```

Comment:
nit - the indentation of the newlines is different for each of these comments. i think first indentation level for issuer is the correct one